crates/pera-core/src/execution_cache/cache_types.rs

```diff
@@ -113,12 +113,6 @@ impl<I: Iterator> AssertOrdered<I> {
     }
 }
 
-impl<I: IntoIterator> From<I> for AssertOrdered<I::IntoIter> {
```

Comment:
This is needed because this implementation conflicts with another implementation from the cryptography-private repository

Reply:
No, there is no point to comment in places where code was removed.